success.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+if (isset($_SESSION['success'])) : ?>
+<div>
+        <h3>
+            <?php
+            echo $_SESSION['success'];
+            unset($_SESSION['success']);
+            ?>
+
+        </h3>
+</div>
+<?php endif ?>
```

Comment:
I don't like the whole purpose of this file. Let's sync on how to do it when you get to this.